CMakeLists.txt

```diff
@@ -110,7 +110,10 @@ target_link_libraries(${PROJECT_NAME}
   ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
   ${QT_LIBRARIES} ${catkin_LIBRARIES} ${Boost_LIBRARIES} log4cxx)
 
-install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
+add_executable(${PROJECT_NAME}_updater src/collisions_updater.cpp )
```

Comment:
This was the original name. Since catkin collects all cmake target there is recommendation of prefixing alls targets with the package name.
If it is okay for you I can rename it.